src/role-access/containers/add-exclusion/search-person/search-person.component.ts

```diff
@@ -54,4 +53,8 @@ export class SearchPersonComponent implements OnInit {
   public selectedPerson(person?: Person) {
```

Comment:
Why is person optional?

Reply:
And no return type

Reply:
It was already optional. I'll see if changing it has any ill effect. Will add return type